docs/user/tutorials/install_epinio_on_rancher_desktop.md

```diff
@@ -0,0 +1,37 @@
+# Rancher Desktop configuration
+
+## Rancher Desktop Prerequisites
+
+* Running on Windows requires Windows Subsystem for Linux (WSL) which is automatically installed by Rancher Desktop
+
+### Install Rancher Desktop
+
+Install the [latest version](https://github.com/rancher-sandbox/rancher-desktop/releases) from Rancher Desktop for your operating system
+
+## Setup Kubernetes
+
+When running Rancher Desktop for the first time wait until the initialization is completed. Make sure that a supported Kubernetes version is selected under `Kubernetes Settings`, e.g. **v1.21.5**
+
+## Install epinio
+
+Make sure Rancher Desktop is running.
+
+Since there might be cases where Rancher Desktop reports Kubernetes as running but some pods are not yet ready the state can be manually verified by executing in a terminal `kubectl get pods -A` and veryfing that all pods report the status `Running` or `Completed`.
+
+### Windows
+
+1. Start a terminal (e.g. type `cmd` in the search field) and change to the directory where `epinio-windows-amd64` is located, e.g. `cd Downloads`
+
+2. Run `epinion-windows-amd64 install`
+
+3. To allow Epinio being executed directly from any terminal copy the binary to a directory of your choice and add it to the `PATH` variable as described [here](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)
```

Comment:
```suggestion
3. To allow Epinio being executed directly from any terminal copy the binary to a directory of your choice and add it to the `PATH` variable as described by [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)
```

Attribution, and IMHO nicer phrasing. I try to avoid `here` links.

Reply:
```suggestion
3. Copy the binary to a directory of your choice and add it to the `PATH` variable as described by [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53). This allows execution of Epinio directly from any terminal.
```

As the first suggesstion, and split the long sentence.